fitbenchmarking/parsing/fitbenchmark_parser.py

```diff
@@ -444,8 +442,7 @@ def _parse_function_value(value: str) -> 'int | float | bool | str':
                 continue
         return value
 
-    @staticmethod
-    def _get_data_points(data_file_path: str):
+    def _get_data_points(self, data_file_path: str):
```

Comment:
I think this can stay static and bal can be made static? Not a big deal though, so happy to keep it not static for future flexibility.

Reply:
I think I remember why I made the changes. I was seeing an arguments differ error in linting. And then I checked if the method needs to be static (i.e. is it being used anywhere else in the other modules), I did not find the usage elsewhere. And also `bal` can be made `@static` but `horace` can't.

<img width="1086" alt="Screenshot 2024-08-13 at 13 39 17" src="https://github.com/user-attachments/assets/07f5d71f-55e0-4ee6-b8cb-c6590fbd2abd">

Reply:
Sorry, yes you are right. I missed the Horace one.